drc_cmis/admin.py

```diff
@@ -37,11 +84,33 @@ class CMISConfigAdmin(SingletonModelAdmin):
         ),
     )
 
-    def cmis_enabled(self, obj):
-        return settings.CMIS_ENABLED
+    class Media:
+        js = ("drc_cmis/js/cmis_config.js",)
+
+    def get_urls(self):
+        urls = super().get_urls()
+        extra_urls = [
+            path(
+                "api/connection",
+                self.admin_site.admin_view(self.cmis_connection_view),
+                name="cmisconfig_api_connection",
+            )
+        ]
+        return extra_urls + urls
+
+    def cmis_connection_view(self, request):
+        return CMISConnectionJSONView.as_view(model_admin=self)(request)
+
+    def cmis_enabled(self, obj=None):
+        return getattr(settings, "CMIS_ENABLED", True)
 
     cmis_enabled.short_description = _("Enabled")
     cmis_enabled.boolean = True
 
+    def cmis_connection(self, obj=None):
```

Comment:
parameter obj is unused

Reply:
Unused but required by the Django admin to use as "field".